### PR TITLE
Fd 33 b

### PR DESCRIPTION
**Pull request by iaroslavm**
My version of token generator

### CODE REVIEW

File: ferrilata-devilline/Services/Interfaces/ITokenService.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+using Microsoft.IdentityModel.Tokens;
+
+namespace ferrilata_devilline.Services.Interfaces
+{
+    public interface ITokenService
+    {
+        string GenerateToken(string UserEmail);
+    }
+
+
```
Comment (sabyanna) [inline ferrilata-devilline/Services/Interfaces/ITokenService.cs:11]: 10 - too many empty lines

File: ferrilata-devilline/Startup.cs [hunk 2]
```diff
@@ -48,6 +51,27 @@ public void ConfigureServices(IServiceCollection services)
             $"database={Environment.GetEnvironmentVariable("FDDATABASE")}; " +
             $"user={Environment.GetEnvironmentVariable("FDUSERNAME")};" +
             $" password={Environment.GetEnvironmentVariable("FDPASSWORD")}"));
+
```
Comment (sabyanna) [inline ferrilata-devilline/Startup.cs]: 54-72 - this is needed for the validation, not for generating tokens

**Review by sabyanna — CHANGES_REQUESTED** [review]

**Review by sabyanna — APPROVED** [review]

File: ferrilata-devilline/Migrations/20190710090246_Test1.cs [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+        {
+
+        }
+
+        protected override void Down(MigrationBuilder migrationBuilder)
+        {
+
+        }
+    }
+}
```
Comment (drexeverest) [inline ferrilata-devilline/Migrations/20190710090246_Test1.cs]: You can remove empty migration files

File: ferrilata-devilline/Services/TokenService.cs [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    public class TokenService : ITokenService
+    {
+        public string GenerateToken(string UserEmail)
+        {
+            var tokenHandler = new JwtSecurityTokenHandler();
+            var key = Encoding.ASCII.GetBytes(Environment.GetEnvironmentVariable("FDTOKENSECRET"));
+
+            var tokenDescriptor = new SecurityTokenDescriptor
+            {
+                Subject = new ClaimsIdentity(new Claim[]
```
Comment (drexeverest) [inline ferrilata-devilline/Services/TokenService.cs:31]: The indentation is not the best here

**Review by drexeverest — CHANGES_REQUESTED** [review]

**Review by AlexTeper — APPROVED** [review]

**Pull request merged**